### ISSUE TITLE
Caption in tabs not fit (design)

### ISSUE THREAD

**Issue by sunnyphp** [ansible-semaphore/semaphore#1451]
Hello.

Russian language, caption in tabs not fit:

![image](https://github.com/ansible-semaphore/semaphore/assets/3903438/68d335c6-9db2-4982-ae6b-6ee90de1107e)

Words:
```text
Task = Задача
Build = Сборка
Deploy = Развертывать
Run = Запуск
```

I propose to change design of the window:

![image](https://github.com/ansible-semaphore/semaphore/assets/3903438/f61f276b-02a9-4087-970d-7c02dafda179)

Thanks.

**Comment by fiftin** [comment]
@sunnyphp Да, спасибо, исправлю ;)

**Comment by aaronnad** [comment]
Similar issue with #1465 with the UI not scaling

**Comment by fiftin** [comment]
Yes, this form require redesign. But now I just replaced Russian -> English. Fixed after 2.9.25.
